.prettierrc.js

```diff
@@ -2,4 +2,5 @@
 
 module.exports = {
   singleQuote: true,
+  extends: 'recommended',
```

Comment:
Is this a valid `Prettier` option? I haven't seen this before and I can't seem to find it in the docs.

Reply:
https://github.com/emberjs/ember-qunit/pull/990